### PR TITLE
[SCB-1660] change deploy.sh format

### PR DESCRIPTION
**Pull request by kakulisen**
scripts/deploy.sh, now the format is dos, will execute fail in Linux
I change format to unix and skip the version that is not snapshot

### CODE REVIEW

**Comment by chanjarster** [comment]
Do you mean the CR/LF ?

**Comment by kakulisen** [comment]
Yes

**Comment by chanjarster** [comment]
LGTM

File: scripts/deploy.sh [hunk 1]
```diff
@@ -1,28 +1,39 @@
...
+##
+## Unless required by applicable law or agreed to in writing, software
+## distributed under the License is distributed on an "AS IS" BASIS,
+## WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+## See the License for the specific language governing permissions and
+## limitations under the License.
+## ---------------------------------------------------------------------------
+#bin/sh
+
+TAGGEDCOMMIT=$(git tag -l --contains HEAD)
```
Comment (WillemJiang) [inline scripts/deploy.sh]: I don't think we need to add this part.
It part is for check if the commit is tagged.

**Comment by kakulisen** [comment]
In my understanding, when there is a tagged commit, it must be a release, so I skipped deploying it.

**Comment by WillemJiang** [comment]
> In my understanding, when there is a tagged commit, it must be a release, so I skipped deploying it.

As we don't release the kit on travis  by this script, my suggestion is removing this part of code.

**Comment by kakulisen** [comment]
OK, I will remove it in the next commit

**Pull request merged**